src/async-components/views/dialogs/keybackup/CreateKeyBackupDialog.js

```diff
@@ -294,14 +294,21 @@ export default React.createClass({
     _renderPhasePassPhraseConfirm: function() {
         const AccessibleButton = sdk.getComponent('elements.AccessibleButton');
 
+        let matchText;
+        if (this.state.passPhraseConfirm === this.state.passPhrase) {
+            matchText = _t("That matches!");
+        } else if (!this.state.passPhrase.startsWith(this.state.passPhraseConfirm)) {
+            // only tell them they're wrong if they've actually gone wrong.
+            // Security concious readers will note that if you left riot-web unattended
+            // on this screen, this would make it easy for a malicious person to guess
+            // your passphrase one letter at a time, but they could get this faster by
+            // just opening the browser's developer tools and reading it.
+            // Note that this includes not having typed anything at all.
```

Comment:
I think you mean we **will not** show a message when you haven't typed anything, but this comment as written makes me think you're saying we would enter this branch when you haven't typed anything.

Reply:
Yeah, it did read like that - thanks!